pkg/proc/bininfo.go

```diff
@@ -1803,6 +1806,9 @@ func (bi *BinaryInfo) setGStructOffsetElf(image *Image, exe *elf.File, wg *sync.
 	case elf.EM_PPC64:
 		_ = getSymbol(image, bi.logger, exe, "runtime.tls_g")
 
+	case elf.EM_RISCV:
```

Comment:
change the case above to be `case elf.EM_PPC64, elf.EM_RISCV:`

Reply:
Done.